assets/src/story-embed-block/block/block.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "web-stories/embed",
+  "category": "embed",
+  "attributes": {
```

Comment:
If it's not a dynamic block then these attributes could use a source instead of being saved into comments.

Reply:
Made it a partially dynamic block now, keeping the `save()` function as a cached copy.